### PR TITLE
Type errors

### PR DESCRIPTION
**Pull request by angelo-v**
I tried to work arround the type errors that occured on master lately.

### CODE REVIEW

File: src/create/create.ts [hunk 1]
```diff
@@ -88,6 +88,7 @@ export function newThingUI (
           _context => {
             const newPaneOptions: NewPaneOptions = Object.assign({
               newBase: newBase,
+              folder: options.folder || undefined,
```
Comment (angelo-v) [inline src/create/create.ts:91]: should folder be optional in NewPaneOptions?

File: src/create/create.ts [hunk 2]
```diff
@@ -209,7 +210,7 @@ export function newThingUI (
   const mintingPanes = Object.values(thePanes).filter(pane => pane.mintNew)
   const mintingClassMap = mintingPanes.reduce((classMap, pane) => {
     if (pane.mintClass) {
-      classMap[pane.mintClass] = (classMap[pane.mintClass] || 0) + 1
+      classMap[pane.mintClass.uri] = (classMap[pane.mintClass.uri] || 0) + 1
```
Comment (angelo-v) [inline src/create/create.ts:213]: Since this seems to be couting by class, using the URI is sufficient I guess

File: src/matrix/matrix.ts [hunk 3]
```diff
@@ -71,7 +71,7 @@ export function matrixForQuery (
     header.setAttribute('style', 'padding: 0.3em;')
     header.textContent = utils.label(y1) // first approximation
     if (y1.termType === 'NamedNode') {
-      kb.fetcher.nowOrWhenFetched(y1.uri.split('#')[0], undefined, function (
+      kb.fetcher!.nowOrWhenFetched(y1.uri.split('#')[0], undefined, function (
```
Comment (angelo-v) [inline src/matrix/matrix.ts:74]: Can we rely on that the fetcher is available. If so, can we reflect that in the type?

Comment (timbl) [reply src/matrix/matrix.ts:74]: Yes.  There is a subclass of Store, LiveStore, (IIRC) which has fetcher and updater.

Comment (angelo-v) [reply src/matrix/matrix.ts:74]: So, should the type in solid-logic changed to LiveStore @michielbdejong

**Pull request merged**